### DISCUSSION TITLE
leetcode-js/blog/principle

### DISCUSSION THREAD

**Discussion by giscus** [2xiao/leetcode-js#4] [Announcements]
# leetcode-js/blog/principle

如何做人生的重要决策 最近经历了一个很重要的人生选择，和男朋友开始异国，打算之后出国留学或者找工作，自此之后的人生又揭开了一层迷雾。 其实做决定的时候并没有过多的纠结，因为我有一个想都不用想的选项，感觉一切都是水到渠成的。但是面对人生的重要节点，soul-searching 的过程是免不了的，所以也在这里记录一下，指导我做 life choice 的一些信念。

https://2xiao.github.io/leetcode-js/blog/principle.html

**Comment by botsp** [comment]
这几条都很有共鸣，但也正是很有共鸣也会困惑，为什么我们都在成年后逐渐认为"人生就是真的很苦", 这个想法真的很中式了、或者说是这一代人的中式思维。
